bundle/bundle.go

```diff
@@ -191,6 +192,12 @@ func (b Bundle) Validate() error {
 		return errors.New("'latest' is not a valid bundle version")
 	}
 
+	for _, requiredExtension := range b.RequiredExtensions {
```

Comment:
I'm actually slightly unsure if this logic matches up to:
> The requiredExtensions array SHOULD contain the EXTENSION NAME defined in the custom object for each extension that is required.

as mentioned in https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md#required-extensions

I'm wondering how a required extension would be utilized successfully if not defined in the `Custom` section?

Reply:
Yeah, I think this check is enough to validate that the items in `requiredExtensions` are defined in `custom`.